ContextualMenuDecorator/ContextualMenuDecorator.module.less

```diff
@@ -33,6 +33,10 @@
 			max-height: @sand-contextualmenu-innercontainer-max-height;
 		}
 
+		&.auto {
```

Comment:
at line 29, suggested change

![image](https://user-images.githubusercontent.com/63335068/177976739-807a1421-389b-4864-a6b9-0e99a1592f67.png)

In this way, the specificity is increased without setting a new css min-width for the "auto" case. Please remove the added min-width and the css variable